src/Components/Skills.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from 'styled-components/macro';
```

Comment:
What does `/macro` do? Does it enable more functionality or is it the same as 
`import styled from 'styled component' `?

Reply:
If you add macro to the end it will be easier to follow the code when inspecting it in the console since you can see the name of the component instead of random figures that don't make sense.  Perhaps it makes more sense if you try it out and inspect your own page. (it might be good for other things that I don't know. )